proto/pocket/shared/service.proto

```diff
@@ -89,3 +97,12 @@ message ConfigOption {
   string value = 2; // Config option value
 }
 
+// ServiceUsageMetrics tracks cumulative (monotonically increasing) onchain service usage
+// - Tracks total # of relays serviced
+// - Tracks total # of compute units consumed
+// - Updated during claim settlement
+message ServiceUsageMetrics {
+  string service_id = 1;
+  uint64 total_relays = 2;
+  uint64 total_compute_units = 3;
```

Comment:
Let's add a `TODO_TECHDEBT` to account for overflows one day in the distant future.
![Screenshot 2025-06-20 at 3 03 53 PM](https://github.com/user-attachments/assets/f033e55d-2e8b-4062-b247-6499e1d1ade4)